cli/tests/integrations/test_config.py

```diff
@@ -151,6 +151,15 @@ def test_set_new_output(env):
     config_set('core.dcos_url', 'http://dcos.snakeoil.mesosphere.com', env)
 
 
+def test_set_nonexistent_subcommand(env):
+    assert_command(
+        ['dcos', 'config', 'set', 'foo.bar', 'baz'],
+        stdout=b'',
+        stderr=b"'foo' is not a dcos command.\n",
+        returncode=1,
+        env=env)
```

Comment:
Great. How about a test for what this bug is actually fixing. The config file has an entry for "missing.something" but the "missing" section/subcommand doesn't exists.